core/contracts/financial_templates/ExpiringMultiParty.sol

```diff
@@ -12,21 +12,27 @@ contract ExpiringMultiParty is Liquidatable {
         uint _positionExpiry,
         uint _positionWithdrawalLiveness,
         address _collateralCurrency,
+        FixedPoint.Unsigned memory _collateralRequirement,
         FixedPoint.Unsigned memory _disputeBondPct,
         FixedPoint.Unsigned memory _sponsorDisputeRewardPct,
         FixedPoint.Unsigned memory _disputerDisputeRewardPct,
-        uint _liquidationLiveness
+        uint _liquidationLiveness,
+        address _finderAddress,
+        bytes32 _priceFeedIdentifier
     )
         public
         Liquidatable(
```

Comment:
What do you think about the tradeoffs of having ExpiringMultiParty? Is this contract even needed? just curious now that you've worked with the contracts

@mrice32 @ptare

Reply:
I dont think we need it in its current form. However I think that @mrice32 is going to create a parent contract which will sit at the top of the three levels for fee collection.

Reply:
I think this is something we can worry about once we have the contract working since it's more of a superficial thing.